### PR TITLE
Updates to README.md

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22723)

### CODE REVIEW

File: README.md
Comment (jedieaston) [inline README.md:55]:
```suggestion
manifests\<first lower case letter of publisher>\<publisher>\<package>\<version>\
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Now it's actually a path.

**Review by jedieaston — APPROVED** [review]

**Comment by vedantmgoyal2009** [comment]
Why is it not merging? Does it need to have the `Azure-Pipeline-Passed` and `Validation-Completed` labels? It's not a manifest...

**Comment by jedieaston** [comment]
It requires manual review from someone with write access.

@KevinLaMS, can you take a look?

**Comment by vedantmgoyal2009** [comment]
@jedieaston The note written [here](https://github.com/microsoft/winget-pkgs#using-the-yamlcreateps1) means that we should copy the script and run it where the manifests are to be created. But, we run the script in the _**Tools**_ folder, thereby neither copying it anywhere else. I think it should be changed.

**Comment by jedieaston** [comment]
I don't think that's been updated since YAMLCreate.ps1 got smarter, so yeah, if you can change that it'd be great.

**Comment by vedantmgoyal2009** [comment]
Well, I cannot think of any other sentence but it can be removed in order to avoid confusion for new contributors.

When I was new to this repository, I read the documentation and had copied the script where the manifests should be created. After that, I got to know that it has to be run from the same folder without copying it anywhere. That day, the whole directory structure corrupted the clone and I had to start over again. 😂

**Review by denelon — APPROVED** [review]

**Comment by denelon** [comment]
Thanks for helping with documentation!!!

**Pull request merged**
